### DISCUSSION TITLE
[TreeTable] collision of "resizableColumns" and "scrollable" props

### DISCUSSION THREAD

**Discussion by TeoDar** [primefaces/community#286] [PrimeVue]
https://github.com/primefaces/community/assets/40923004/201cd86c-4c5a-4c35-8dfa-3ab6799cb7c1

Good day PrimeVue team.
Please tell me if this is a bug or not? When you try to make TreeTable scrollable and allow resize columns, the width of the columns is not updated  when opening the tree, but resets each time to the preset one.

Can I somehow update the width of the columns when I click on the expand tree element button?

**Comment by tugcekucukoglu** [comment]
Thanks for the screen record!

I've tried the reproduce the issue using this stackblitz https://stackblitz.com/edit/flbaqa?file=src%2FApp.vue%3AL5. It seems it's working correctly. How can I replicate the issue?

**Reply by TeoDar** [reply]
Good day.

I opened your stackblitz link and the problem is still the same.
When adding "Scrollable" prop and changing the column width, when re-opening the element in the tree, the column width in the element is reset to the original.


https://github.com/primefaces/community/assets/40923004/ee4a8c28-33d6-41e0-97d1-bbe416d39870

**Reply by TeoDar** [reply]
@cagataycivici Maybe you could give me some advice?

**Comment by TeoDar** [comment]
UP
